[2025-07-23T01:56:03.228+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.fetch_people_in_space manual__2025-07-23T01:55:56.578672+00:00 [queued]>
[2025-07-23T01:56:03.242+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.fetch_people_in_space manual__2025-07-23T01:55:56.578672+00:00 [queued]>
[2025-07-23T01:56:03.243+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-07-23T01:56:03.263+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): fetch_people_in_space> on 2025-07-23 01:55:56.578672+00:00
[2025-07-23T01:56:03.269+0000] {standard_task_runner.py:60} INFO - Started process 303 to run task
[2025-07-23T01:56:03.273+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'fetch_people_in_space', 'manual__2025-07-23T01:55:56.578672+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/main_pipeline.py', '--cfg-path', '/tmp/tmpp4i2okev']
[2025-07-23T01:56:03.277+0000] {standard_task_runner.py:88} INFO - Job 8: Subtask fetch_people_in_space
[2025-07-23T01:56:03.361+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_pipeline.fetch_people_in_space manual__2025-07-23T01:55:56.578672+00:00 [running]> on host d113f261e9bc
[2025-07-23T01:56:03.474+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ingrid' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='fetch_people_in_space' AIRFLOW_CTX_EXECUTION_DATE='2025-07-23T01:55:56.578672+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-23T01:55:56.578672+00:00'
[2025-07-23T01:56:03.787+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-07-23T01:56:03.803+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=fetch_people_in_space, execution_date=20250723T015556, start_date=20250723T015603, end_date=20250723T015603
[2025-07-23T01:56:03.849+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-07-23T01:56:03.898+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
